### PR TITLE
Testing Code

### PR DESCRIPTION
**Pull request by Brenaren**
#4 - Wrote unit tests for existing classes that are testable outside of just running the program. There could always be more tests added to catch wierd edge cases, but these cover the basics of the classses.
In order to compile the code users will now have to make sure the google test suite is installed on their linux machine. The instructions are in the README.md under install

### CODE REVIEW

File: tests/Login_unit_tests.cpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+#include "../include/Login.hpp"
+
+namespace {
+
+TEST(Login, createUser) {
+    /* Test creation of users with valid and invalid input. */
+    Login login = Login();
+    EXPECT_EQ(login.createUser("Jon", "12345", "owner"), true);
+    EXPECT_EQ(login.createUser("Jon", "12345", "Baconator"), false);
```
Comment (nshoap-utk) [inline tests/Login_unit_tests.cpp:11]: Where's the beef?

Comment (Brenaren) [reply tests/Login_unit_tests.cpp:11]: I know I know, I didn't get my beef in there... I apologize and we can fix it in a future update.

**Review by nshoap-utk — APPROVED** [review]
Good job!

**Review by josephwehby — APPROVED** [review]
tested on my machine and everything worked. looks good!
